cnf-certification-test/webserver/index.js

```diff
@@ -211,4 +215,164 @@ var field='<label for="'+clickedId+counters[clickedId] +'">'+counters[clickedId]
+  document.getElementById('access-control').checked = true
+  document.getElementById('platform-alteration').checked = true
+  document.getElementById('networking').checked = true
+  document.getElementById('performance').checked = true
+  document.getElementById('observability').checked = true
+
+ } 
+
+function selectScenario(table){
+  var idtable = '#'+table+'-table'
+  $(idtable).empty()
+ // addCheckedTest()
+  var field = ""
+  const selectScenarioComboBox = document.getElementById('selectScenarioComboBox')
+  const selectedValue = selectScenarioComboBox.options[selectScenarioComboBox.selectedIndex].value
+  if (selectedValue === 'all') {
+    document.getElementById('selectOpt').setAttribute('hidden','hidden')
+
+  for (const key in classification) {
+    if (key.startsWith(table)){
+
+      field=
+      '<rh-accordion><rh-accordion-header><div>'+    
+      '<input type="checkbox" id ="'+key+ '" value="'+
+      key +'" name="selectedOptions" checked> ' +
+      '<label for="'+key+ '">'+key+'</label>'+
+      '</div></rh-accordion-header>'+
+      '<rh-accordion-panel><h4>Description </h4><p>'+
+      classification[key][0].description+'</p>'+
+      '<h4>Remediation</h4><p>'+
+      classification[key][0].remediation+'</p>'+
+      '<h4>BestPracticeReference</h4><p>'+
+      classification[key][0].bestPracticeReference+'</p></rh-accordion-panel>'+
+      '</rh-accordion>'
+      $(field).appendTo(idtable);
+    }
+  }
+
+  }else{ 
+    document.getElementById('selectOpt').removeAttribute('hidden')
+
+  buildTest(selectedValue,table)
+  }
+
+}
+function buildTest(scenarioValue,table){
+  var idtable = '#'+table+'-table'
+  const option = document.getElementById('selectOpt')
+  const selectedValue = option.options[option.selectedIndex].value
+  console.log(selectedValue)
+  var field = ""
+for (const key in classification){
+  eqData = classification[key][0]["categoryClassification"].FarEdge
+
+  if (scenarioValue == "telco") {
+    eqData = classification[key][0]["categoryClassification"].Telco
+  }
+  if (scenarioValue == "nontelco") {
+    eqData = classification[key][0]["categoryClassification"].NonTelco
+  }
+  if (scenarioValue == "extended") {
+    eqData = classification[key][0]["categoryClassification"].Extended
+  }
+  if(eqData == selectedValue){
+    if (key.startsWith(table)){
+    field=
+    '<rh-accordion><rh-accordion-header><div>'+    
+    '<input type="checkbox" id ="'+key+ '" value="'+
+    key +'" name="selectedOptions" checked> ' +
+    '<label for="'+key+ '">'+key+'</label>'+
+    '</div></rh-accordion-header>'+
+    '<rh-accordion-panel><h4>Description </h4><p>'+
+    classification[key][0].description+'</p>'+
+    '<h4>Remediation</h4><p>'+
+    classification[key][0].remediation+'</p>'+
+    '<h4>BestPracticeReference</h4><p>'+
+    classification[key][0].bestPracticeReference+'</p></rh-accordion-panel>'+
+    '</rh-accordion>'
+     $(field).appendTo(idtable);
```

Comment:
Disable click propagation here by adding this code:
     document.getElementById(key).addEventListener('click', function(event) {
      // Prevent click propagation from the input element
      event.stopPropagation();
    });